### ISSUE TITLE
Error: ?ASlink-Warning-Undefined Global 'cpct_fw2hw_asm'

### ISSUE THREAD

**Issue by enrib4** [lronaldo/cpctelera#127]
SDCC does not allow compilation in assembly of the "cpct_fw2hw" function, trying to declare it as:

`.globl cpct_fw2hw_asm`

Error in the linker:

```
?ASlink-Warning-Undefined Global 'cpct_fw2hw_asm' referenced by module 'cpct_keyboard'

?ASlink-Warning-Undefined Global 'cpct_fw2hw_asm' referenced by module ''
/home/.../cpctelera//cfg/global_main_makefile.mk:66: fallo en las instrucciones para el objetivo 'obj/game.ihx'
```

**Comment by SpDizzy** [comment]
An easy fix would be declare it as global, as you are doing:

`.globl cpct_fw2hw_asm `

And to add a **double colon** on it's assembly entry point, on **cpct_fw2hw_asmbindings.s**

Like this:

`cpct_fw2hw_asm::`
 
It looks like original definition has just one semicolon, not beign global thus.

**Comment by lronaldo** [comment]
Thanks @enrib4 for this issue report. In fact, as @SpDizzy says, is just a mistake with the assembly label. It should have been marked as global, but it was marked as local. 

Now it's fixed (commit 507e20f4df5ae0b854a47a7feae973b3f5db488a).

Thanks both of you :)
